Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1549,7 +1550,7 @@ def prepare_RAW2RECO(self, sequence = None):
     def prepare_RAW2DIGI(self, sequence = "RawToDigi"):
         self.loadDefaultOrSpecifiedCFF(sequence,self.RAW2DIGIDefaultCFF)
         self.scheduleSequence(sequence,'raw2digi_step')
-        #	    if self._options.isRepacked:
+         #          if self._options.isRepacked:
```

Comment:
Another unnecessary white-space change.

Reply:
If it is not strictly necessary I would not update the code now so that CMSSW_10_3_0 can be built. Either me or Slava could make a new PR later with this change and the update of the modifier.

Reply:
I couldn't figure out how to undo it in my editor (since a while I set it up to use spaces instead of tabs).
So, this will need either an attempt with a different editor or some more complicated rewrite of the full PR set of commits.

Reply:
what you see is the best I managed to do yesterday while undoing a much larger unnecessary edit in these lines.